cfgov/cal/event.py

```diff
@@ -1,10 +1,12 @@
-from cal.models import CFPBCalendar, CFPBCalendarEvent
-from datetime import date, datetime
+from cal.models import CFPBCalendarEvent
+from datetime import datetime
+import re
```

Comment:
Nitpicky, and not a PEP8 thing I suppose, but it'd be nice to follow [PEP8 import guidelines](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:
 1. standard library imports
 2. related third party imports
 3. local application/library specific imports

 You should put a blank line between each group of imports.

So, alternatively:

```py
import re

from datetime import datetime

from cal.models import CFPBCalendarEvent
```